modules/ossm-config-sampling.adoc

```diff
@@ -9,20 +9,20 @@ The distributed tracing sampling rate is set to sample 100% of traces in your se
 
 A trace is an execution path between services in the service mesh. A trace is comprised of one or more spans. A span is a logical unit of work that has a name, start time, and duration.
 
-The sampling rate determines how often a trace is generated. Configure sampling as a scaled integer representing 0.01% increments. 
+The sampling rate determines how often a trace is generated. Configure sampling as a scaled integer representing 0.01% increments.
 
 In a basic installation, `spec.tracing.sampling` is set to `10000`, which samples 100% of traces. For example:
 
-* Setting the value to 10 samples 0.1% of traces. 
-* Setting the value to 500 samples 5% of traces. 
+* Setting the value to 10 samples 0.1% of traces.
+* Setting the value to 500 samples 5% of traces.
 
 Setting the value to `10000` is useful for debugging, but can affect performance. For production, set `spec.tracing.sampling` to `100.`
 
 .Procedure
 
 . In the {product-title} web console, click *Operators* -> *Installed Operators*.
 
-. From the *Project* menu, select the project where you installed the control plane, for example `istio-system`.
+. Click the *Project* menu and select the project where you installed the control plane, for example `istio-system`.
```

Comment:
```suggestion
. Click the *Project* menu and select the project where you installed the control plane, for example *istio-system*.
```
In this context, it's a UI item and should be enclosed in asterisks. Everywhere else, it's a unique name for an API resource and should be enclosed in backticks. This chart is handy: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

Reply:
Replaced in 10 files.